backend/app/controllers/products.js

```diff
@@ -500,8 +501,14 @@ module.exports = {
                 formattedExportRow.stage = rowStage.title;
                 formattedExportRow.question = exportData.body[i].questionText;
                 formattedExportRow.questionType = exportData.body[i].questionType;
+                formattedExportRow.questionIndex = exportData.body[i].questionIndex;
```

Comment:
So we get index `0` for the first question. Should we `+1` here so the indexes start at `1` and not `0`?

Reply:
@katebrittain  do you want 0- or 1-based indexing for question numbers. Sadly, probably we want 1-based indexing to match the client ui.